utils/nvme.go

```diff
@@ -131,8 +131,8 @@ func (n *Nvme) ListFeatures(device string) ([]byte, error) {
 }
 
 // nolint:gocyclo // line parsing is cyclomatic
-func (n *Nvme) parseNvmeFeatures(d *nvmeDeviceAttributes) ([]nvmeDeviceFeatures, error) {
-	out, err := n.ListFeatures(d.DevicePath)
+func (n *Nvme) ParseNvmeFeatures(device string) ([]nvmeDeviceFeatures, error) {
+	out, err := n.ListFeatures(device)
```

Comment:
suggestion: name the `device` parameter `devicePath` instead?

Reply:
This has been updated everywhere.